src/components/MeasurementOverlay/MeasurementOverlay.js

```diff
@@ -108,8 +108,14 @@ class MeasurementOverlay extends React.PureComponent {
 
   isMouseInsideRect = (e, overlayElement) => {
     const overlayRect = overlayElement.getBoundingClientRect();
-    const { clientX: x, clientY: y } = e;
-
+    let x,y;
+    if (e instanceof TouchEvent && e.touches) {
```

Comment:
@ZhijieZhang please review this part. it seems, when in mobile, event type is not of type mouse event and is of type touch event. hence that is why the measurement overlay does not fade when you are drawing under it.

Reply:
I see. This change makes sense and looks good to me.

Reply:
@achenPDFTron Nice job! Verified that the fading works.